charts/node/templates/service.yaml

```diff
@@ -114,7 +114,7 @@ spec:
     {{- if $.Values.node.perNodeServices.relayP2pService.ws.enabled }}
     - name: ws
       port: {{ $.Values.node.perNodeServices.relayP2pService.ws.port | int }}
-      targetPort: 30334
+      targetPort: 30444
```

Comment:
I would set it to the opposite, it should be 29444. When we deploy a parachain node, the relay port is "secondary" and honestly few people would care about it.

Reply:
Agree, but for non-parachain nodes, the relaychain would be primary chain. see https://github.com/paritytech/helm-charts/issues/178#issuecomment-1273418120, please. let me know about the final decision.

Reply:
final decision is to leave it aside for now and use `30334` for p2p ports other than relay-tcp.